enhancements/two-node-fencing/tnf.md

```diff
@@ -175,72 +181,84 @@ Three aspects of cluster creation need to happen for a vanilla two-node cluster
 
 ### API Extensions
 
 The full list of changes proposed is [summarized above](#summary-of-changes). Each change is broken down in greater detail below.
 
 #### Feature Gate Changes
 
-We will define a new `DualReplicaTopology` feature that can be enabled in `install-config.yaml` to ensure the clusters running this feature cannot be upgraded.
+We will define a new `DualReplicaTopology` feature that can be enabled in `install-config.yaml` to ensure the clusters running this feature cannot be upgraded until the feature is ready for general
+availability.
 
 #### Infrastructure API Changes
 
-A mechanism is needed for components of the cluster to understand that this is a two-node control-plane topology that may require different handling.
-We will define a new value for the `infrastructureTopology` field of the Infrastructure config's `TopologyMode` enum.
-Specifically, the value of `DualReplica` will be added to the currently supported list, which includes `HighlyAvailable`, `SingleReplica`, and `External`.
+A mechanism is needed for components of the cluster to understand that this is a two-node control-plane topology that may require different handling. We will define a new value for the
+`infrastructureTopology` field of the Infrastructure config's `TopologyMode` enum. Specifically, the value of `DualReplica` will be added to the currently supported list, which includes
+`HighlyAvailable`, `SingleReplica`, and `External`.
 
-InfrastructureTopology is assumed to be immutable by components of the cluster. While there is strong interest in changing this to allow for topology
-transitions to and from TNF, this is beyond the scope of this enhancement proposal and should be detailed in its own enhancement.
+InfrastructureTopology is assumed to be immutable by components of the cluster. While there is strong interest in changing this to allow for topology transitions to and from TNF, this is beyond the
+scope of this enhancement proposal and should be detailed in its own enhancement.
 
 #### etcd Operator Changes
 
-Initially, the creation of an etcd cluster will be driven in the same way as other platforms.
-Once the cluster has two members, the etcd daemon will be removed from the static pod definition and recreated as a resource controlled by RHEL-HA.
-At this point, the cluster-etcd-operator (CEO) will be made aware of this change so that some membership management functionality that is now handled by RHEL-HA can be disabled.
-This will be achieved by having the same entity that drives the configuration of RHEL-HA use the OpenShift API to update a field in the CEO's `ConfigMap` - which can only succeed if the control-plane is healthy.
+Initially, the creation of an etcd cluster will be driven in the same way as other platforms. Once the cluster has two members, the etcd daemon will be removed from the static pod definition and
```

Comment:
I need to review this paragraph again when I'm more familiar with CEO...

Reply:
As with some of the others, I think it would be OK to update this in a separate PR. The goal from my side was to make sure there was a section where folks could write and adjust as part of their own investigations.